### ISSUE TITLE
stript payment 

### ISSUE THREAD

**Issue by dhorajiya9898** [jonasbark/flutter_stripe_payment#349]
flutter_stripe_payment did not work i don't know why

**Comment by SERDARPINAR** [comment]
Definitely. and all the sample codes are intertwined. The examples show 3 different packages and they are all unrelated.
